### ISSUE TITLE
Server Error (status code 500)

### ISSUE THREAD

**Issue by bnodir** [Azure-Samples/azure-search-openai-demo#1390]
> Please provide us with the following information:
> ---------------------------------------------------------------

### This issue is for a: (mark with an `x`)
```
- [x] bug report -> please search issues before submitting
- [ ] feature request
- [ ] documentation issue or request
- [ ] regression (a behavior that used to work and stopped in a new release)
```

### Minimal steps to reproduce
>

### Any log messages given by the failure
>
Error: The server had an error while processing your request. Sorry about that! You can retry your request, or contact us through an Azure support request at: https://go.microsoft.com/fwlink/?linkid=2213926 if the error persists. (Please include the request ID 6d6321f8-e4aa-4a77-9aa0-67868802d7ca in your message.) { "error": { "message": "The server had an error while processing your request. Sorry about that! You can retry your request, or contact us through an Azure support request at: https://go.microsoft.com/fwlink/?linkid=2213926 if the error persists. (Please include the request ID 6d6321f8-e4aa-4a77-9aa0-67868802d7ca in your message.)", "type": "server_error", "param": null, "code": null } } 500 {'error': {'message': 'The server had an error while processing your request. Sorry about that! You can retry your request, or contact us through an Azure support request at: https://go.microsoft.com/fwlink/?linkid=2213926 if the error persists. (Please include the request ID 6d6321f8-e4aa-4a77-9aa0-67868802d7ca in your message.)', 'type': 'server_error', 'param': None, 'code': None}} <CIMultiDictProxy('Content-Length': '412', 'Content-Type': 'application/json', 'Access-Control-Allow-Origin': '*', 'x-content-type-options': 'nosniff', 'x-ratelimit-remaining-requests': '19', 'apim-request-id': 'e325ff30-3455-4f6c-b180-9abebe5bae5d', 'x-ratelimit-remaining-tokens': '19996', 'x-request-id': '6d6321f8-e4aa-4a77-9aa0-67868802d7ca', 'ms-azureml-model-error-reason': 'model_error', 'ms-azureml-model-error-statuscode': '500', 'x-ms-client-request-id': 'Not-Set', 'x-ms-region': 'East US 2', 'azureml-model-session': 'wc2931f89ed18-6', 'Strict-Transport-Security': 'max-age=31536000; includeSubDomains; preload', 'Date': 'Fri, 08 Mar 2024 00:57:23 GMT')>

### Expected/desired behavior
>

### OS and Version?
> Windows 7, 8 or 10. Linux (which distribution). macOS (Yosemite? El Capitan? Sierra?)

### azd version?
> run `azd version` and copy paste here.

### Versions
>

### Mention any other details that might be useful

> ---------------------------------------------------------------
> Thanks! We'll be in touch soon.

**Comment by pamelafox** [comment]
Is this after running "azd up" and visiting the app? Did you try to file a support request?

**Comment by bnodir** [comment]
> Is this after running "azd up" and visiting the app?
Yes, I tried it. I had no problems after waiting for a few hours, so I will close the issue now.

> Did you try filing a support request?
No, not yet.

Not sure if it is related to the issue, but I recently attempted the following connection with the existing Azure OpenAI that I already have. Right now, both systems work fine.
https://stackoverflow.com/questions/76931791/connecting-excel-vba-to-azure-openai
